tests/test-dirs/no-escape.t/run.t

```diff
@@ -240,7 +240,7 @@ Syntax errors also shouldn't escape:
           "line": 1,
           "col": 41
         },
-        "type": "parser",
+        "type": "typer",
```

Comment:
Is there a good reason for this change?

Reply:
I didn't dig into this, but I assumed that there was an flambda change that caused this error to be reported by typing instead of parsing.